jib-maven-plugin/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to this project will be documented in this file.
 ## [unreleased]
 
 ### Added
+- Additionally reads credentials from `.docker/.dockerconfigjson` and legacy Docker config (`.docker/.dockercfg`). Also searches for `$HOME/.docker` in (addition to current `System.get("user.home")`). This may help retrieve credentials, for example, on Kubernetes. (#[2260](https://github.com/GoogleContainerTools/jib/issues/2260))
```

Comment:
```suggestion
- Additionally reads credentials from `.docker/.dockerconfigjson` and legacy Docker config (`.docker/.dockercfg`). Also searches for `$HOME/.docker` in (addition to current `System.get("user.home")`). This may help retrieve credentials, for example, on Kubernetes. ([#2260](https://github.com/GoogleContainerTools/jib/issues/2260))
```